mm2src/coins/nft/nft_structs.rs

```diff
@@ -293,4 +364,39 @@ pub(crate) struct NftTransferHistoryWrapper {
 #[derive(Debug, Serialize)]
 pub struct NftsTransferHistoryList {
     pub(crate) transfer_history: Vec<NftTransferHistory>,
+    pub(crate) skipped: usize,
+    pub(crate) total: usize,
+}
+
+#[allow(dead_code)]
+#[derive(Clone, Debug, Deserialize)]
+pub struct NftTxHistoryFilters {
+    #[serde(default)]
+    pub(crate) receive: bool,
+    #[serde(default)]
+    pub(crate) send: bool,
```

Comment:
I chose these parameters because of GUI.
![image](https://github.com/KomodoPlatform/atomicDEX-API/assets/52405288/f0cec75e-7b4d-4d0a-b464-00fb80ebe2e3)
So if both are false(or true) then I just ignore it and return all transfers. 
```rust
if filters.send && !filters.receive {
            sql_builder.sql_builder().and_where_eq("status", "'Send'");
        } else if filters.receive && !filters.send {
            sql_builder.sql_builder().and_where_eq("status", "'Receive'");
        }
```

Reply:
I guess we can leave it as is since this is what the GUI team wants